[2021-11-27 09:46:11,085] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: elt.upload scheduled__2021-11-27T02:45:00+00:00 [queued]>
[2021-11-27 09:46:11,092] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: elt.upload scheduled__2021-11-27T02:45:00+00:00 [queued]>
[2021-11-27 09:46:11,092] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 09:46:11,092] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-11-27 09:46:11,092] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 09:46:11,101] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): upload> on 2021-11-27 02:45:00+00:00
[2021-11-27 09:46:11,105] {standard_task_runner.py:52} INFO - Started process 4774 to run task
[2021-11-27 09:46:11,111] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'elt', 'upload', 'scheduled__2021-11-27T02:45:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/processing.py', '--cfg-path', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpp1lgsjb6', '--error-file', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpq3gq384q']
[2021-11-27 09:46:11,114] {standard_task_runner.py:77} INFO - Job 219: Subtask upload
[2021-11-27 09:46:11,151] {logging_mixin.py:109} INFO - Running <TaskInstance: elt.upload scheduled__2021-11-27T02:45:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-11-27 09:46:11,187] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=elt
AIRFLOW_CTX_TASK_ID=upload
AIRFLOW_CTX_EXECUTION_DATE=2021-11-27T02:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-27T02:45:00+00:00
[2021-11-27 09:46:11,188] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T
[2021-11-27 09:46:11,189] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/Users/Home/Airflow/tasks/cleaning.py /Users/Home/Airflow/output worldbank']
[2021-11-27 09:46:11,196] {subprocess.py:85} INFO - Output:
[2021-11-27 09:46:11,261] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.IRIG.AG.ZS_DS2_en_csv_v2_3163914.csv: No such file or directory
[2021-11-27 09:46:11,266] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_AG.LND.AGRI.ZS_DS2_en_csv_v2_3163149.csv: No such file or directory
[2021-11-27 09:46:11,267] {subprocess.py:89} INFO - API_AG.LND.IRIG.AG.ZS_DS2_en_csv_v2_3163914.csv
[2021-11-27 09:46:11,267] {subprocess.py:89} INFO - API_AG.LND.AGRI.ZS_DS2_en_csv_v2_3163149.csv
[2021-11-27 09:46:11,271] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-27 09:46:11,289] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=elt, task_id=upload, execution_date=20211127T024500, start_date=20211127T024611, end_date=20211127T024611
[2021-11-27 09:46:11,333] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-27 09:46:11,351] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
